plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarPluginImpl.kt

```diff
@@ -108,6 +108,8 @@ open class ScaleBarPluginImpl(
   override fun onPluginView(view: View) {
     scaleBar = view as? ScaleBar
       ?: throw IllegalArgumentException("The provided view needs to implement ScaleBarContract.ScaleBarView")
+    val pixelRatio = mapTransformDelegate.getMapOptions().pixelRatio
+    scaleBar.pixelRatio = pixelRatio
```

Comment:
shall we add the test to the pixelRatio?

Reply:
And nit: we can get the pixelRatio from 
```
override fun bind(mapView: FrameLayout, attrs: AttributeSet?, pixelRatio: Float): View 
```
in this class, no need to fetch it again from mapTransformDelegate.